scripts/do-release.sh

```diff
@@ -6,8 +6,17 @@ set -e
 . "${DAPPER_SOURCE}/scripts/lib/image_defs"
 # shellcheck source=scripts/lib/utils
 . "${DAPPER_SOURCE}/scripts/lib/utils"
+# shellcheck disable=SC1091
 . "${SCRIPTS_DIR}/lib/debug_functions"
 
+### Constants ###
+
+readonly SOURCE_BUNDLE_DIR=${SOURCE_BUNDLE_DIR:-"projects/submariner-operator/packagemanifests"}
+readonly TARGET_BUNDLE_DIR=${TARGET_BUNDLE_DIR:-"projects/community-operators/operators/submariner"}
+readonly TARGET_ORG_UPSTREAM=${TARGET_ORG_UPSTREAM:-"k8s-operatorhub"}
+readonly TARGET_REPO_UPSTREAM=${TARGET_REPO_UPSTREAM:-"${TARGET_ORG_UPSTREAM}/community-operators"}
+readonly PR_TEMPLATE="https://raw.githubusercontent.com/${TARGET_REPO_UPSTREAM}/main/docs/pull_request_template.md"
```

Comment:
Since these are all related only to bundles, I think it's best to prefix them with BUNDLE or something of the sort, eg:
```suggestion
readonly BUNDLE_SOURCE_DIR=${BUNDLE_SOURCE_DIR:-"projects/submariner-operator/packagemanifests"}
readonly BUNDLE_TARGET_DIR=${BUNDLE_TARGET_DIR:-"projects/community-operators/operators/submariner"}
readonly BUNDLE_TARGET_ORG=${BUNDLE_TARGET_ORG:-"k8s-operatorhub"}
readonly BUNDLE_TARGET_REPO=${BUNDLE_TARGET_REPO:-"${BUNDLE_TARGET_ORG}/community-operators"}
readonly BUNDLE_PR_TEMPLATE="https://raw.githubusercontent.com/${BUNDLE_TARGET_REPO}/main/docs/pull_request_template.md"
```